<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowextensions</class>
 <widget class="QDialog" name="windowextensions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration Partie</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Qlabel_informations">
       <property name="text">
        <string>Informations pour la Partie</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="QLabel_nb_joueurs">
         <property name="text">
          <string>Nombre de Joueurs : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="QSpinBox_nb_joueurs">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Extensions :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="QCheckBox_riviere">
         <property name="text">
          <string>Rivière</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="QcheckBox_auberges">
         <property name="text">
          <string>Auberges &amp; Cathédrale</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="QCheckBox_abbes">
         <property name="text">
          <string>Abbes</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="QcheckBox_paysan">
         <property name="text">
          <string>Paysan</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>windowextensions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
